src/java/maskImageWriter/MaskImageWriter.java

```diff
@@ -0,0 +1,53 @@
+package maskImageWriter;
+
+import lombok.Data;
+import map.FloatMask;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.*;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+
+@Data
+public class MaskImageWriter {
+    private FloatMask redMask;
+    private FloatMask greenMask;
+    private FloatMask blueMask;
+
+    public MaskImageWriter(FloatMask redMask, FloatMask greenMask, FloatMask blueMask, float scaleMultiplier, String path) throws IOException {
```

Comment:
The constructor shouldn't be used to write the image. Most likely this can be a static method of the class and the instance variables are not needed. Or there needs to be a proper constructor and then a separate write image method for the class